autox/informer.go

```diff
@@ -19,26 +23,26 @@ func newIter8Watcher(resourceTypes []schema.GroupVersionResource, namespaces []s
 	}
 	// for each namespace, resource type configure Informer
 	for _, ns := range namespaces {
-		w.factories[ns] = dynamicinformer.NewFilteredDynamicSharedInformerFactory(k8sclient.dynamicClient, 0, ns, nil)
+		w.factories[ns] = dynamicinformer.NewFilteredDynamicSharedInformerFactory(Client.dynamicClient, 0, ns, nil)
```

Comment:
Should `Client` be local? If it is not used outside this package, this should be local.